### ISSUE TITLE
Should we retain AdGuard branding for downstream uses?

### ISSUE THREAD

**Issue by mcatanzaro** [AdguardTeam/BlockYouTubeAdsShortcut#11]
Hi, I hope you don't mind if I abuse your issue tracker for this query. [We're planning to use your script for Epiphany's adblocker](https://gitlab.gnome.org/GNOME/epiphany/-/merge_requests/1000) because it works with no changes and is licensed GPLv3+ (thanks!). However, since it adds the text "with AdGuard" next to the YouTube logo, I wonder if you would prefer us to (a) do nothing and retain the AdGuard branding, or (b) debrand it. (a) gets you a little free advertising, I suppose. (b) ensures Epiphany users won't think AdGuard is to blame if the script breaks in the future, or for other problems with Epiphany's adblocker (it's just [EasyList in content blocker form](https://easylist-downloads.adblockplus.org/easylist_min_content_blocker.json), which doesn't seem to be maintained super well).

If we don't hear back soon, we'll probably include it with your branding intact. Anyway, happy to respect your preferences here. Thanks again for making this open source!

**Comment by ameshkov** [comment]
@mcatanzaro Hi Michael, I guess retaining AG branding may confuse people, they don't expect to see AdGuard there. Does Epiphany adblocker has a switch somewhere in the browser settings? It'd be ideal if you could mention somewhere near that switch. If this is not possible, it'd be enough just to mention AdGuard in the place where you list all other dependencies.

> it's just EasyList in content blocker form, which doesn't seem to be maintained super well

Consider using [this](https://github.com/AdguardTeam/SafariConverterLib/) instead. It's LGPL at this moment, but we're going to change it to GPLv3.

**Comment by janbrummer** [comment]
Within the current MR it is internal and there is no user option where we can add some information. This means the only documentation is within source code.

**Comment by Exalm** [comment]
I mean we have the general ad blocker switch in preferences.

**Comment by janbrummer** [comment]
Yes, but it's for the complete list (depending on the filter rule set) and not just AdGuard.

**Comment by ameshkov** [comment]
No worries, if there's no place where AG can be mentioned in the UX, just use it, keeping branding is not necessary.

**Comment by ameshkov** [comment]
> Thing is we don't want to run our own infrastructure to build or host the adblock rules

It is supposed to be used on the client-side, not on the server-side. Makes it possible to use any lists you need and convert them in runtime.

**Comment by mcatanzaro** [comment]
I'm afraid we'd have to rewrite it all to avoid depending on Swift. But thanks for the suggestion anyway.
